pkg/tfgen/generate.go

```diff
@@ -1254,7 +1254,14 @@ func (g *Generator) gatherResource(rawname string,
 
 		// TODO[pulumi/pulumi#397]: represent sensitive types using a Secret<T> type.
 		doc, foundInAttributes := getDescriptionFromParsedDocs(entityDocs, key)
-		rawdoc := propschema.Description()
+		rawdoc, elided := reformatText(infoContext{
+			language: g.language,
+			pkg:      g.pkg,
+			info:     g.info,
+		}, propschema.Description(), nil)
```

Comment:
This puts description based docs through our general docs cleaning pipeline, including the java change.